### PR TITLE
Updating loc hooks for strings and "Touch Develop" references

### PR DESCRIPTION
**Pull request by millerv**

### CODE REVIEW

**Comment by millerv** [comment]
@pelikhan - could you review this when convenient?  Thanks!

File: lib/CloudSessions.ts [hunk 1]
```diff
@@ -416,26 +416,26 @@ module TDev.RT {
...
             else
-                m.addOk("ok");
+                m.addOk(lf("ok"));
 
             var retrytimeindicatorinterval = setInterval(() => {
                 var msg: string;
                 var mr = session.user_get_missing_rounds();
                 if (mr) {
-                    msg = mr + " remaining."
+                    msg = mr + lf(" remaining.")
```
Comment (pelikhan) [inline lib/CloudSessions.ts]: Do ``lf("%0 remaining.", mr)``

File: lib/CloudSessions.ts [hunk 2]
```diff
@@ -416,26 +416,26 @@ module TDev.RT {
...
                 }
                 var rt = session.user_get_next_connection_attempt();
                 if (rt) {
                     var secs = Math.floor((rt - 700 - new Date().getTime()) / 1000);
-                    var msg = (secs < 1) ? "connection attempt in progress..." :
-                        "next connection attempt in " + secs.toString() + " seconds";
+                    var msg = (secs < 1) ? lf("connection attempt in progress...") :
+                        lf("next connection attempt in ") + secs.toString() + lf(" seconds");
```
Comment (pelikhan) [inline lib/CloudSessions.ts]: Same: ``lf("next connection attempt in %0 seconds")``

**Comment by millerv** [comment]
@pelikhan Updated lines 432 and 437 per your recommendations.

**Pull request merged**